contentcuration/contentcuration/tests/test_file_storage.py

```diff
@@ -1,73 +0,0 @@
-import pytest
```

Comment:
Half baked test that I committed. Used to test minio. Not needed anymore, since other parts of the code use minio and object storage now.